script/job_generator/generate_job_yaml.py

```diff
@@ -290,8 +305,10 @@ def job_variables(job: Dict[str, Tuple[str, str]]) -> Dict[str, str]:
     else:
         variables["ALPAKA_TEST_MDSPAN"] = "OFF"
 
+    # enabled backend depending on the job parameters
```

Comment:
Do we need this comment here again?

Reply:
No, I believe it was a left over, before I moved the code to an extra function.